src/gui/plugins/scene3d/Scene3D.hh

```diff
@@ -161,6 +169,22 @@ inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
     private: bool OnViewCollisions(const msgs::StringMsg &_msg,
         msgs::Boolean &_res);
 
+    /// \brief Get the text for the popup error message
+    /// \return The error text
+    public: Q_INVOKABLE QString ErrorPopupText() const;
+
+    /// \brief Set the text for the popup error message
+    /// \param[in] _errorTxt The error text
+    public: Q_INVOKABLE void SetErrorPopupText(const QString &_errorTxt);
+
+    /// \brief Notify the popup error text has changed
+    signals: void ErrorPopupTextChanged();
+
+    /// \brief Notify that an error has occurred (opens popup)
+    /// Note that the function name needs to start with lowercase in order for
+    /// the connection to work on the QML side 
```

Comment:
Note to self, remove [Scene3D.hh:185](https://github.com/ignitionrobotics/ign-gazebo/blob/38893b969b0a300f548a68f19e9437e2328d9125/src/gui/plugins/scene3d/Scene3D.hh#L185) line ending whitespace. Don't want to take up CI so waiting to see if there is anymore feedback.

Reply:
2d6a9301443ab06d3fb217ee22be9254781389b6